src/main/scala/zio/intellij/inspections/simplifications/SimplifySucceedInspection.scala

```diff
@@ -0,0 +1,23 @@
+package zio.intellij.inspections.simplifications
+
+import org.jetbrains.plugins.scala.codeInspection.collections._
+import org.jetbrains.plugins.scala.lang.psi.api.expr.ScExpression
+import zio.intellij.inspections._
+import zio.intellij.inspections.zioMethods._
+import zio.intellij.utils.StringUtils._
+
+class SimplifySucceedInspection extends ZInspection(SucceedSimplificationType)
+
+object SucceedSimplificationType extends SimplificationType {
```

Comment:
Why not to make it part of SimplifyZipInspection? For me it is mostly simplification for zip, not succeed

Reply:
Makes sense — I moved code there.